src/functions/locale.rs

```diff
@@ -24,17 +27,15 @@ pub fn set_locale(locale: String) {
         files::append_file("/mnt/etc/locale.gen", "en_US.UTF-8 UTF-8\n"),
```

Comment:
this extra `\n` can be removed here since append_file adds newlines automatically